docs/developer-guide/building.md

```diff
@@ -183,60 +183,78 @@ components from `./bits` using the `./deps/eng/tools/bits-upload.sh` script.
      in your environment. If not set, `$UPDATES_IMGADM_CHANNEL` is computed
      automatically. See `./deps/eng/tools/bits-upload.sh`
 
-### Building from Gerrit
+### Useful development tools and building from GitHub Pull Requests
 
-When reviewing proposed changes from Gerrit, it can be useful to build and
-deploy those directly.
+Setting up the following tools is likely to make your work on Manta and Triton
+a little easier:
 
-Here we build patch set 3 of the `sdc-sapi.git` component, which has the
-gerrit id `5538`:
+* [hub](https://hub.github.com/) is a command line extension to git, with
+  particularly useful commands to list, checkout and create pull requests.
+* [jr](https://github.com/joyent/joyent-repos) is a tool that makes it
+  easier to interact with the many Joyent repositories you'll be working with
+  when developing Manta and Triton. It allows you to list and clone
+  repositories, as well as allowing you to run commands that affect multiple
+  repositories.
+* [prr](https://github.com/joyent/prr/) is a command line tool for merging an
+  approved pull request, allowing you to modify the commit message.
+
+When reviewing proposed changes from GitHub, it can be useful to build and
+deploy those directly allowing reviewers to exercise the changes before they're
+integrated.
+
+In this example, we'll use `jr` to clone the
+[sdc-manta](https://github.com/joyent/sdc-manta) repository, then we'll
+use `hub` to get the changes from the
+[joyent/sdc-manta#21](https://github.com/joyent/sdc-manta/pull/21) pull
+request.
+
+First clone the repository:
+
+```
+-bash-4.3$ jr clone sdc-manta
+Clone 1 repo into "/tmp"? [y/N] y
+cloned "sdc-manta" to "/tmp/sdc-manta.git" (3s)
+-bash-4.3$ cd sdc-manta.git
+```
+
+Now use `hub` to obtain the changes from the pull request and build them:
 
 ```
--bash-4.1$ cd /tmp
--bash-4.1$ cd sdc-sapi
--bash-4.1$ git ls-remote | grep 5538
-From https://cr.joyent.us/joyent/sdc-sapi.git
-d2daf78578e3854069cbe194f5f9cf4c96571d22        refs/changes/38/5538/1
-53f51b8e1b4e6088b22757ee230edc9b6974e46e        refs/changes/38/5538/2
-2dc9ec5ca8ef57c73126b572e3683adde86793bd        refs/changes/38/5538/3
--bash-4.1$ git fetch origin refs/changes/38/5538/3
-From https://cr.joyent.us/joyent/sdc-sapi
- * branch            refs/changes/38/5538/3 -> FETCH_HEAD
--bash-4.1$ git checkout -b cr5538-3 FETCH_HEAD
-Switched to a new branch 'cr5538-3'
--bash-4.1$ git describe --all --long
-heads/cr5538-3-0-g2dc9ec5
--bash-4.1$ make all release publish buildimage
+-bash-4.3$ hub pr checkout 21
```

Comment:
Would a more likely first example be making a change and creating a new PR via `hub pull-request` ?

Reply:
It would if we were explaining how hub works, but the intent of this section was how to build other people's code review requests.

I'll re-title it to "Useful development tools and dealing with GitHub Pull Requests"

then add a simple example of creating a PR

Reply:
Ah sorry, I missed the context. Thanks.